.github/workflows/linters.yml

```diff
@@ -16,7 +16,11 @@ jobs:
           node-version: "12.x"
       - name: Setup ESLint
         run: |
+<<<<<<< HEAD
+          npm install --save-dev eslint@6.8.x eslint-config-airbnb@18.1.x eslint-plugin-import@2.20.x eslint-plugin-jsx-a11y@6.2.x eslint-plugin-react@7.20.x eslint-plugin-react-hooks@2.5.x babel-eslint@10.1.x
+=======
           npm install --save-dev eslint@6.8.x eslint-config-airbnb@18.1.x eslint-plugin-import@2.22.x eslint-plugin-jsx-a11y@6.2.x eslint-plugin-react@7.20.x eslint-plugin-react-hooks@2.5.x babel-eslint@10.1.x
+>>>>>>> 14c27a5ba3d6756fb99ef0d2b64d9764a3a19a24
```

Comment:
Well done guys,

I think you need to look into your `github/workflows`.
Perhaps this issue is caused by merge conflicts.

Your milestone 1;
![image](https://user-images.githubusercontent.com/31897434/101850313-d4029800-3b59-11eb-97d0-5c9f67254ca8.png)

Your milestone 2 even thou you submitted this as milestone 4 🤦 ;
![image](https://user-images.githubusercontent.com/31897434/101850460-14621600-3b5a-11eb-9be4-8c34796b1834.png)